ctypesgen/printer_python/preamble.py

```diff
@@ -417,10 +62,13 @@ def __call__(self, *args):
 
 def ord_if_char(value):
```

Comment:
I think it might make sense to remove this function, too. In my case, it's put in at places where it doesn't make sense, and all the repeated instance checks perhaps have a performance impact.
IMO, it would be much nicer to just let the user call `ord()` where it's desired.

Example (macro):
```python
# fpdf_edit.h: 15
def FPDF_ARGB(a, r, g, b):
    return (uint32_t (ord_if_char((((((uint32_t (ord_if_char(b))).value & 0xff) | (((uint32_t (ord_if_char(g))).value & 0xff) << 8)) | (((uint32_t (ord_if_char(r))).value & 0xff) << 16)) | (((uint32_t (ord_if_char(a))).value & 0xff) << 24))))).value
```

Reply:
Done. New example macro:
```python
# fpdf_edit.h: 15
def FPDF_ARGB(a, r, g, b):
    return uint32_t(((((uint32_t(b).value & 0xff) | ((uint32_t(g).value & 0xff) << 8)) | ((uint32_t(r).value & 0xff) << 16)) | ((uint32_t(a).value & 0xff) << 24))).value
```

Reply:
There are still some unnecessary layers of brackets I didn't manage to get rid of...